data-prepper-plugins/otel-logs-source/build.gradle

```diff
@@ -31,7 +31,6 @@ dependencies {
     implementation "org.bouncycastle:bcpkix-jdk15on:1.69"
     testImplementation 'org.assertj:assertj-core:3.24.2'
     testImplementation testLibs.mockito.inline
-    testImplementation "org.hamcrest:hamcrest:2.2"
```

Comment:
Is this dependency not used?

Reply:
This is defined in the root `build.gradle` and all projects get it. So it doesn't need to be declared here.

Reply:
To elaborate:

It is declared here: https://github.com/opensearch-project/data-prepper/blob/298e7931aa3b26130048ac3bde260e066857df54/build.gradle#LL90C16-L90C16

Since it is inside the `subprojects` section all sub-projects get it. https://github.com/opensearch-project/data-prepper/blob/298e7931aa3b26130048ac3bde260e066857df54/build.gradle#L65